bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+
+import os
+import sys
+import subprocess
+import argparse
+import json
+
+# TODO
+# We are using external apps just for the moment.
+# These will be removed for native libraries.
+AWSCLIENT = "/usr/local/bin/aws"
+PGCLIENT = "/usr/bin/psql"
+
+# NZSL: Is there a database url defined in the environment?
+DATABASE_URL = os.getenv("DATABASE_URL", None)
+if not DATABASE_URL:
+    print("You must define DATABASE_URL in the environment.", file=sys.stderr)
+    exit()
+
+# AWS: Is there an AWS_PROFILE defined in the environment?
+AWS_PROFILE = os.getenv("AWS_PROFILE", None)
+if not AWS_PROFILE:
+    print(
+        "You must define AWS_PROFILE in the environment. Eg. AWS_PROFILE='nzsl'",
+        file=sys.stderr,
+    )
+    exit()
+
+parser = argparse.ArgumentParser(
+    description="You must define, in the environment: AWS_PROFILE, DATABASE_URL"
+)
+
+# Optional arguments
+parser.add_argument(
+    "--mode",
+    default="uat",
+    required=False,
+    help="Mode to run in, eg 'production, 'uat', etc (default: '%(default)s')",
+)
+parser.add_argument(
+    "--cached",
+    default=False,
+    required=False,
+    action="store_true",
+    help="Use keys generated on a previous non-cached run (default: %(default)s) "
+    "(Don't mix PRODUCTION and STAGING!)",
+)
+parser.add_argument(
+    "--pgclient",
+    default=PGCLIENT,
+    required=False,
+    help=f"Postgres client path (default: %(default)s)",
+)
+parser.add_argument(
+    "--awsclient",
+    default=AWSCLIENT,
+    required=False,
+    help=f"AWS client path (default: %(default)s)",
+)
+args = parser.parse_args()
+
+NZSL_APP = f"nzsl-signbank-{args.mode}"
```

Comment:
issue: it looks like we only use this in output, so I think we should axe it (we can restore it later if we actually need it)

Reply:
Aye.

Reply:
Done.